draft-ietf-teep-protocol.md

```diff
@@ -2126,6 +2126,13 @@ bz/m4rVlnIXbwK07HypLbAmBMcCjbazR14vTgdzfsJwFLbM5kdtzOLSolg==
 ## Example 3: Supplying Personalization Data for Trusted Component Binary {#suit-personalization}
 {: numbered='no'}
 
+This example uses the following parameters:
+
+- Algorithm for payload encryption: AES-GCM-128
```

Comment:
I think this is obsolete now based on #356 and #373?

Reply:
As noted in my comments to the issue, I think that the use of AES-GCM is correct.

Reply:
Sent email to the list to discuss, since AES-GCM is not in suit-mti.